changeEclipse.sh

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
any reasons to let an empty first line?

Reply:
any reason not to have an empty first line?

Reply:
it's just the first time that I see that so i'm curious if it is a good practice i'm not aware of.
I would think that less code is better.